askama_derive/src/parser/expr.rs

```diff
@@ -194,6 +194,7 @@ enum Suffix<'a> {
     Attr(&'a str),
     Index(Expr<'a>),
     Call(Vec<Expr<'a>>),
+    MacroCall(&'a str),
```

Comment:
What is the value of the unit variant here representing? Seems strange that it isn't a `Vec` of something like for `Call`.

Reply:
It's the whole content, everything in `(…)`. Proc macros can access the input verbatim including any white spaces, so I don't think we should not attempt to understand what's going on in a macro invocation, just paste it verbatim.

Reply:
It's representing the inner body/args of the macro call. It's possible Vec of Expr would work there, but given that it can accept potentially any kind of syntax (valid rust or not), it's probably the safer bet to just pass the args as is.